examples/transparent.rs

```diff
@@ -2,6 +2,7 @@
 // Copyright 2021-2023 Tauri Programme within The Commons Conservancy
 // SPDX-License-Identifier: Apache-2.0
 
+#[cfg(windows)]
 use std::{num::NonZeroU32, rc::Rc};
```

Comment:
Note that there are many more "unused import" warnings like this: why the CI not catching those?